packages/waggle/src/socket/listeners/connection.ts

```diff
@@ -3,11 +3,17 @@ import { CertsData, IChannelInfo, IMessage } from '../../common/types'
 import IOProxy from '../IOProxy'
 import PeerId from 'peer-id'
 import logger from '../../logger'
+import { EventEmitter } from 'events'
 const log = logger('socket')
 
-export const connections = (io, ioProxy: IOProxy) => {
+export const connections = (io, ioProxy: IOProxy, connectionManager: EventEmitter) => {
```

Comment:
There is no need to pass connectionsManager as separate argument because ioproxy instance already has reference to the connectionsManager